services/blockchain-indexer/shared/database/schema/accounts.js

```diff
@@ -17,21 +17,19 @@ module.exports = {
 	primaryKey: 'address',
 	schema: {
 		address: { type: 'string' },
```

Comment:
Could you please update the tables as per the recommendations in branch `1106-index-all-transactions-for-dpos-module`?

Reply:
I updated account table and the rest will merge with the branch `1106-index-all-transactions-for-dpos-module`